R/quotes_linter.R

```diff
@@ -0,0 +1,84 @@
+#' Character string quote linter
+#'
+#' Check that the desired quoting character is used to delimit string constants.
+#'
+#' @param quote_char Which quote character to accept. Defaults to the tidyverse
```

Comment:
I am wondering if `delimiter` might be a better parameter name here than `quote_char`? 🤔

Reply:
good idea -- that's the term used in ?Quotes

![screen shot of ?Quotes text esp. "Single and double quotes delimit character constants."](https://user-images.githubusercontent.com/7606389/226722256-1e389a9c-e1c7-449d-b46b-fcc0f80a1bd5.jpg)

Reply:
Done